specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2025-03-01-preview/Issues.json

```diff
@@ -0,0 +1,1248 @@
+        ]
+      },
+      "readOnly": true
+    },
+    "FetchInvestigationResultParameters": {
+      "type": "object",
+      "description": "Parameters provided to get the investigation result",
+      "properties": {
+        "investigationId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The unique identifier of the investigation"
+        }
+      },
+      "required": [
+        "investigationId"
+      ]
+    },
+    "InvestigationExecution": {
+      "type": "object",
+      "description": "Details about the execution of the investigation",
+      "properties": {
+        "runState": {
+          "$ref": "#/definitions/InvestigationExecutionRunState",
+          "description": "The state of the investigation execution"
+        },
+        "completedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time at which the investigation execution completed"
+        }
+      },
+      "required": [
+        "runState",
+        "completedAt"
+      ]
+    },
+    "InvestigationExecutionRunState": {
+      "type": "string",
+      "description": "The execution state of the investigation",
+      "enum": [
+        "None",
+        "InProgress",
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "InvestigationExecutionRunState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "None",
+            "value": "None",
+            "description": "The investigation execution didn't start"
+          },
+          {
+            "name": "InProgress",
+            "value": "InProgress",
+            "description": "The investigation execution is in progress"
+          },
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "The investigation execution completed successfully"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "The investigation execution failed"
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "The investigation execution was canceled"
+          }
+        ]
+      },
+      "readOnly": true
+    },
+    "InvestigationMetadata": {
+      "type": "object",
+      "description": "Properties of the current investigation",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The unique identifier of the investigation"
+        },
+        "execution": {
+          "$ref": "#/definitions/InvestigationExecution",
+          "description": "The execution details of the investigation"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The creation time of the investigation"
+        },
+        "runParameters": {
```

Comment:
>runParameters [](http://example.com/codeflow?start=9&length=13)

Private repo has investigationScopes.  are there no customers using the api-version now who will have issues with this new property ?

Reply:
No, no customers are using it now, they will not be impacted